.pre-commit-config.yaml

```diff
@@ -39,11 +39,6 @@ repos:
     hooks:
       - id: tox-ini-fmt
         args: [ "-p", "fix" ]
-  - repo: https://github.com/asottile/setup-cfg-fmt
```

Comment:
Can you add https://github.com/tox-dev/pyproject-fmt instead now?

Reply:
CI failing

Reply:
Will have a look 👍